CONTRIBUTING.md

```diff
@@ -199,7 +199,7 @@ When using OpenShift, the OLM is already installed.
 
 ### Create the bundle and related images
 
-The following commands will generate a bundle under `bundle/`, build an image with its contents, build and publish the operator image.
+The following commands will generate two bundles (one for regular Kubernetes and another one for OpenShift) under `bundle/`, build an image with its contents, build and publish the operator image.
```

Comment:
The follow up command fails 

```
 BUNDLE_IMG=docker.io/pavolloffay/opentelemetry-operator-bundle:latest IMG=docker.io/pavolloffay/opentelemetry-operator:latest make bundle container container-push bundle-build bundle-push                                                                                                                        ploffay@fedora
BUNDLE_VARIANT=community VERSION=0.103.0-13-gb4817246 make generate-bundle
make[1]: Entering directory '/home/ploffay/projects/open-telemetry/opentelemetry-operator'
/home/ploffay/projects/open-telemetry/opentelemetry-operator/bin/controller-gen "crd:generateEmbeddedObjectMeta=true,maxDescLen=0" rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
cd config/manager && /home/ploffay/projects/open-telemetry/opentelemetry-operator/bin/kustomize edit set image controller=docker.io/pavolloffay/opentelemetry-operator:latest
make[2]: Entering directory '/home/ploffay/projects/open-telemetry/opentelemetry-operator'
/home/ploffay/projects/open-telemetry/opentelemetry-operator/bin/controller-gen crd:generateEmbeddedObjectMeta=true,maxDescLen=0,maxDescLen=1200 rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=/tmp/tmp.CyfvCS41bn/bases
make[2]: Leaving directory '/home/ploffay/projects/open-telemetry/opentelemetry-operator'
# Warning: 'patchesStrategicMerge' is deprecated. Please use 'patches' instead. Run 'kustomize edit fix' to update your Kustomization automatically.
grep: warning: stray \ before #
sed -i 's/minKubeVersion: .*/minKubeVersion: 1.23.0/' config/manifests/community/bases/opentelemetry-operator.clusterserviceversion.yaml
/home/ploffay/projects/open-telemetry/opentelemetry-operator/bin/operator-sdk generate kustomize manifests -q --input-dir config/manifests/community --output-dir config/manifests/community
cd ./bundle/community && cp ../../PROJECT . && /home/ploffay/projects/open-telemetry/opentelemetry-operator/bin/kustomize build ../../config/manifests/community | /home/ploffay/projects/open-telemetry/opentelemetry-operator/bin/operator-sdk generate bundle -q --overwrite --version "0.103.0"   --output-dir . --kustomize-dir ../../config/manifests/community && rm PROJECT
# Warning: 'bases' is deprecated. Please use 'resources' instead. Run 'kustomize edit fix' to update your Kustomization automatically.
# Warning: 'patchesStrategicMerge' is deprecated. Please use 'patches' instead. Run 'kustomize edit fix' to update your Kustomization automatically.
# Warning: 'vars' is deprecated. Please use 'replacements' instead. [EXPERIMENTAL] Run 'kustomize edit fix' to update your Kustomization automatically.
# Warning: 'patchesStrategicMerge' is deprecated. Please use 'patches' instead. Run 'kustomize edit fix' to update your Kustomization automatically.
# Warning: 'patchesJson6902' is deprecated. Please use 'patches' instead. Run 'kustomize edit fix' to update your Kustomization automatically.
INFO[0000] Creating bundle.Dockerfile                   
INFO[0000] Creating metadata/annotations.yaml           
INFO[0000] Bundle metadata generated successfully       
# Workaround for https://github.com/operator-framework/operator-sdk/issues/4992
echo "" >> bundle/community/bundle.Dockerfile
echo "LABEL com.redhat.openshift.versions=v4.12" >> bundle/community/bundle.Dockerfile
echo "" >> bundle/community/metadata/annotations.yaml
echo "  com.redhat.openshift.versions: v4.12" >> bundle/community/metadata/annotations.yaml
/home/ploffay/projects/open-telemetry/opentelemetry-operator/bin/operator-sdk bundle validate ./bundle/community
INFO[0000] All validation tests have completed successfully 
./hack/ignore-createdAt-bundle.sh
make[1]: Leaving directory '/home/ploffay/projects/open-telemetry/opentelemetry-operator'
BUNDLE_VARIANT=openshift VERSION=0.103.0-13-gb4817246 make generate-bundle
make[1]: Entering directory '/home/ploffay/projects/open-telemetry/opentelemetry-operator'
/home/ploffay/projects/open-telemetry/opentelemetry-operator/bin/controller-gen "crd:generateEmbeddedObjectMeta=true,maxDescLen=0" rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
cd config/manager && /home/ploffay/projects/open-telemetry/opentelemetry-operator/bin/kustomize edit set image controller=docker.io/pavolloffay/opentelemetry-operator:latest
make[2]: Entering directory '/home/ploffay/projects/open-telemetry/opentelemetry-operator'
/home/ploffay/projects/open-telemetry/opentelemetry-operator/bin/controller-gen crd:generateEmbeddedObjectMeta=true,maxDescLen=0,maxDescLen=1200 rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=/tmp/tmp.3EczcP8rwm/bases
make[2]: Leaving directory '/home/ploffay/projects/open-telemetry/opentelemetry-operator'
# Warning: 'patchesStrategicMerge' is deprecated. Please use 'patches' instead. Run 'kustomize edit fix' to update your Kustomization automatically.
grep: warning: stray \ before #
sed -i 's/minKubeVersion: .*/minKubeVersion: 1.23.0/' config/manifests/openshift/bases/opentelemetry-operator.clusterserviceversion.yaml
/home/ploffay/projects/open-telemetry/opentelemetry-operator/bin/operator-sdk generate kustomize manifests -q --input-dir config/manifests/openshift --output-dir config/manifests/openshift
cd ./bundle/openshift && cp ../../PROJECT . && /home/ploffay/projects/open-telemetry/opentelemetry-operator/bin/kustomize build ../../config/manifests/openshift | /home/ploffay/projects/open-telemetry/opentelemetry-operator/bin/operator-sdk generate bundle -q --overwrite --version "0.103.0"   --output-dir . --kustomize-dir ../../config/manifests/openshift && rm PROJECT
# Warning: 'bases' is deprecated. Please use 'resources' instead. Run 'kustomize edit fix' to update your Kustomization automatically.
# Warning: 'patchesStrategicMerge' is deprecated. Please use 'patches' instead. Run 'kustomize edit fix' to update your Kustomization automatically.
# Warning: 'vars' is deprecated. Please use 'replacements' instead. [EXPERIMENTAL] Run 'kustomize edit fix' to update your Kustomization automatically.
# Warning: 'patchesStrategicMerge' is deprecated. Please use 'patches' instead. Run 'kustomize edit fix' to update your Kustomization automatically.
# Warning: 'patchesJson6902' is deprecated. Please use 'patches' instead. Run 'kustomize edit fix' to update your Kustomization automatically.
INFO[0000] Creating bundle.Dockerfile                   
INFO[0000] Creating metadata/annotations.yaml           
INFO[0000] Bundle metadata generated successfully       
# Workaround for https://github.com/operator-framework/operator-sdk/issues/4992
echo "" >> bundle/openshift/bundle.Dockerfile
echo "LABEL com.redhat.openshift.versions=v4.12" >> bundle/openshift/bundle.Dockerfile
echo "" >> bundle/openshift/metadata/annotations.yaml
echo "  com.redhat.openshift.versions: v4.12" >> bundle/openshift/metadata/annotations.yaml
/home/ploffay/projects/open-telemetry/opentelemetry-operator/bin/operator-sdk bundle validate ./bundle/openshift
INFO[0000] All validation tests have completed successfully 
./hack/ignore-createdAt-bundle.sh
make[1]: Leaving directory '/home/ploffay/projects/open-telemetry/opentelemetry-operator'
/home/ploffay/projects/open-telemetry/opentelemetry-operator/bin/controller-gen object:headerFile="hack/boilerplate.go.txt" paths="./..."
grep: warning: stray \ before #
grep: warning: stray \ before #
grep: warning: stray \ before #
grep: warning: stray \ before #
grep: warning: stray \ before #
grep: warning: stray \ before #
grep: warning: stray \ before #
grep: warning: stray \ before #
grep: warning: stray \ before #
grep: warning: stray \ before #
CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o bin/manager_amd64 -ldflags "-s -w -X github.com/open-telemetry/opentelemetry-operator/internal/version.version=0.103.0-13-gb4817246 -X github.com/open-telemetry/opentelemetry-operator/internal/version.buildDate=2024-07-08T09:20:25Z -X github.com/open-telemetry/opentelemetry-operator/internal/version.otelCol="0.103.1" -X github.com/open-telemetry/opentelemetry-operator/internal/version.targetAllocator=0.103.0 -X github.com/open-telemetry/opentelemetry-operator/internal/version.operatorOpAMPBridge="0.103.0" -X github.com/open-telemetry/opentelemetry-operator/internal/version.autoInstrumentationJava="1.32.1" -X github.com/open-telemetry/opentelemetry-operator/internal/version.autoInstrumentationNodeJS="0.51.0" -X github.com/open-telemetry/opentelemetry-operator/internal/version.autoInstrumentationPython="0.46b0" -X github.com/open-telemetry/opentelemetry-operator/internal/version.autoInstrumentationDotNet="1.2.0" -X github.com/open-telemetry/opentelemetry-operator/internal/version.autoInstrumentationGo="v0.13.0-alpha" -X github.com/open-telemetry/opentelemetry-operator/internal/version.autoInstrumentationApacheHttpd="1.0.4" -X github.com/open-telemetry/opentelemetry-operator/internal/version.autoInstrumentationNginx="1.0.4"" main.go
docker build -t docker.io/pavolloffay/opentelemetry-operator:latest .
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.
[1/2] STEP 1/2: FROM alpine:3.20 AS certificates
[1/2] STEP 2/2: RUN apk --no-cache add ca-certificates
--> Using cache 9d6e8875c3b1cd29a8e3e765e675c089edf08aeae26704c4ed64e1bf32190d5b
--> 9d6e8875c3b1
[2/2] STEP 1/7: FROM scratch
[2/2] STEP 2/7: ARG TARGETARCH
--> Using cache faa128d2e7c033e23eaa727766de37f4b9791b51f239f550a9b5e36c35872381
--> faa128d2e7c0
[2/2] STEP 3/7: WORKDIR /
--> Using cache af19f20e243a4f20cf8ff8afb8681595cf6b2ec1afaa18495dba86acebe49b47
--> af19f20e243a
[2/2] STEP 4/7: COPY --from=certificates /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
--> Using cache 52f6cf493b169301a899f7308b5b5bdec707562374ee8866c5f5e2cda07f60ec
--> 52f6cf493b16
[2/2] STEP 5/7: COPY bin/manager_${TARGETARCH} manager
--> 3e8a500dfc2c
[2/2] STEP 6/7: USER 65532:65532
--> a21e59eee361
[2/2] STEP 7/7: ENTRYPOINT ["/manager"]
[2/2] COMMIT docker.io/pavolloffay/opentelemetry-operator:latest
--> a8b7cc638a30
Successfully tagged docker.io/pavolloffay/opentelemetry-operator:latest
a8b7cc638a30fe0a9d94573bb912fd62c514f6692fecb565506965fe7371babf
docker push docker.io/pavolloffay/opentelemetry-operator:latest
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.
Getting image source signatures
Copying blob c14a68a88537 done   | 
Copying blob f18a14cb113a skipped: already exists  
Copying config a8b7cc638a done   | 
Writing manifest to image destination
docker build -f bundle.Dockerfile -t docker.io/pavolloffay/opentelemetry-operator-bundle:latest .
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.
Error: stat /home/ploffay/projects/open-telemetry/opentelemetry-operator/bundle.Dockerfile: no such file or directory
make: *** [Makefile:543: bundle-build] Error 125
```

Not related to this PR, but it would be great to change `USER` to `DOCKER_USER`

Reply:
Oh! Thanks for the catch!